ecommerce-ui/src/App.js

```diff
@@ -0,0 +1,269 @@
+import './App.css';
+import React, {useState} from "react";
+import VacationRental from './rentals/VacationRental'
+import bnbs from './_data/bnbs.json'
+import ShoppingCart from "./rentals/ShoppingCart";
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+import BnBForm from "./rentals/BnBForm";
+
+
+function App() {
+
+  const [cart, setCart] = useState([])
+  const [cartVisible, setCartVisible] = useState(true)
+  const [formVisible, setFormVisible] = useState(true)
+  const [bnbList, setBnbList] = useState(bnbs)
+  const [title, setTitle]= useState("")
+  const [houseType, sethouseType]= useState("")
+  const [image, setImage]= useState("")
+  const [city, setCity]= useState("")
+  const [country, setCountry]= useState("")
+  const [cost, setCost]= useState("")
+  const [description, setDescription]= useState("")
+  const [name, setName]= useState("")
+  const [isSuperhost, setisSuperhost]= useState(false)
+  const [stars, setStars]= useState("")
+  const [reviews, setReviews]= useState("")
```

Comment:
This would be a good case where you could make an object to hold all of this state, and use one setter to set everything. Example:

```js
const [form, setForm] = useState({
  houseType: '',
  image: '',
  ...
});

const setAttribute = (attribute, value) => {
  setForm({...form, [attribute]: value});
}
```